docs/azure-cni-setup.md

```diff
@@ -0,0 +1,32 @@
+# AMLArc with AzureCNI AKS cluster and Private Link AML Workspace
```

Comment:
it seems here is a guide of how to secure workspace/aks, do we also need a full-guide for others workspace-related resources like blobstorage/acr/keyvault

Reply:
Instructions for these other resources haven't been requested by users yet (to my knowledge), if they are we can expand the docs.